docs/src/administration/users.md

```diff
@@ -104,9 +104,9 @@ title=In the Console
 - Select the project where you want to add a new user.
 - Click {{< icon settings >}} **Settings**.
 - Click **Access**.
-- Expand the user you want to delete.
-- Click **Delete**.
-- Click **Save**.
+- Click the user you want to delete.
+- Click **Remove user**.
+- Click **Accept**.
```

Comment:
🛠️ I believe this should be an ordered list.

Reply:
Definitely should. Didn't look at everything, but that's a quick fix.